### ISSUE TITLE
Need EdgeDeflector Instructions

### ISSUE THREAD

**Issue by Sly-McNasty** [da2x/EdgeDeflector#111]
I removed Edge from my computer, a while back, at least I got rid of most of it.  But I'm finding bits of it everywhere, it's like a cancer.  So I finally found out how to turn off the Screen Protector, and did so to enable me to install EdgeDeflector.  When EdgeDeflector installed, Edge was called on to display the instructions.  At least that how it looked.  I got a flash of an Edge screen and then nothing.  Do I actually need to install Edge to make it so I don't have to use Edge for searches?

**Comment by da2x** [comment]
The installation instruction page is here:
https://www.daniel.priv.no/tools/edgedeflector/post-install.html

However, it only displays the instructions when opened in Edge. It assumes that everything is already set up if the page is opened in any other browser. I need to find a solution to that problem. Here’s a copy of the instructions:

<ol>
      <li>Open the Windows Settings app.</li>
      <li>Go to Apps: Default apps.</li>
      <li>Scroll to the bottom and click Choose default apps by protocol.</li>
      <li>Locate MICROSOFT-EDGE in the list and click on the program icon to the right of it.</li>
      <li>Select EdgeDeflector from the app list. Confirm the change if asked to.</li>
    </ol>

**Comment by Sly-McNasty** [comment]
First, I want to say that was a VERY FAST response, very impressive.  Thank you.  Your instructions were simple, and worked perfectly.  I was a little worried at first because it open up a Bing site, and it was blank.  So I went up to the tool bar of my browser and gave a "Trusted" in my NoScript app and it became an Auslogics.com page.  Then it hit my brain that I could not have done the "Trusted" via the NoScript app if I wasn't on my Firefox browser.  When I was done laughing I started to make this response.  LOL  Thank you very much guy, you're awesome.
